c-api/resvg.h

```diff
@@ -166,6 +166,11 @@ typedef struct resvg_transform {
     double f;
 } resvg_transform;
 
+/**
+ * @brief Returns an identity transform.
+ */
+resvg_transform resvg_transform_default();
```

Comment:
I think we should name it `resvg_transform_identity` to be more clear.

Reply:
I have made the requested change, but the the "default" name was chosen to mirror the same name you chose in tiny_skia.  I don't mind either way...